config/manager/manager.yaml

```diff
@@ -57,6 +57,10 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        - name: POD_NAMESPACE
```

Comment:
- any scenario where `POD_NAMESPACE` will be different to `WATCH_NAMESPACE`, if not we can use the same i guess?

Reply:
I thought that this may be something we might want to support in the future but I agree that this will be the same for now. I will remove WATCH_NAMESPACE.